src/test/java/htsjdk/samtools/sra/SRALazyRecordTest.java

```diff
@@ -32,9 +32,9 @@ public void testSerialization(final SRAAccession accession) throws Exception {
         Assert.assertEquals(deserializedSAMRecord, initialSAMRecord, "Deserialized SAMRecord not equal to original SAMRecord");
     }
 
-    @Test
-    public void testCloneAndEquals() throws Exception {
-        final SRAFileReader reader = new SRAFileReader(DEFAULT_ACCESSION);
+    @Test(dataProvider = "serializationTestData")
```

Comment:
Does this correspond to this PR?

Reply:
no....but I noticed it, so I fixed it...it requires a provider but wasn't given one....